pkg/virt-handler/device-manager/BUILD.bazel

```diff
@@ -33,6 +33,7 @@ go_test(
         "device_controller_test.go",
         "device_manager_suite_test.go",
         "generic_device_test.go",
+        "mediated_device_test.go",
```

Comment:
Before this PR uni tests for mediated devices didn't run at all...

Reply:
lol... i didn't even know this was possible. something new for me to look out for i suppose